code/__DEFINES/stat.dm

```diff
@@ -33,3 +33,8 @@
 #define SHUTTLE_STRANDED 4
 #define SHUTTLE_ESCAPE 5
 #define SHUTTLE_ENDGAME 6
+
+//Movement modes
+#define WALK 0

```

Comment:
I can see if this works, but I probably won't be able to practically tell the difference in terms of performance on a test server